08-transport.Rmd

```diff
@@ -359,11 +393,12 @@ There are two types of transport nodes:
 2. Nodes that are a part of transport networks, representing individual pathways, intersections between pathways (junctions) and points for entering or exiting a transport network such as bus stops and train stations.
 
 From a mathematical perspective transport networks are simply large graphs.
+<!-- not entirely sure what you mean here: are you talking about connecting a node outside of the network by finding the closest node of the graph network? -->
 The first type of node can be connected to the network with "centroid connectors" [@hollander_transport_2016], the geographic location of which can influence model results, by simulating excessive traffic on streets surrounding zone centroids [@jafari_investigation_2015].
```

Comment:
Not entirely sure what you mean here: are you talking about connecting a node outside of the network by finding the closest node of the graph network?